README.md

```diff
@@ -12,7 +12,7 @@ npm install --save saga-maker
 
 ## Usage
 
-To getting started, lets see a basic usage. Let's imaginate we have an online book store. Every time you register a book in system, you need to save in database, and also index this item in ElasticSearch for performatic searches. If for a reason you can't index in ElasticSearch, you need to remove this register from database to avoid inconsistency. You can implementate this using Saga Maker:
+To getting started, lets see a basic usage. Let's imagine we have an online book store. Every time you register a book in system, you need to save in database and also index this item in ElasticSearch for performatic searches. If for some reason the index on ElasticSearch failed and to avoid inconsistency, you need to remove the register from database. You can implementate this using Saga Maker:
```

Comment:
@Johnys In this phrase *"If for some reason the index on ElasticSearch failed..."*, I thought would be more correctly if change *index* to *indexation*. You agree?

My suggestion is:
*If for some reason the indexation on ElasticSearch failed, to avoid inconsistency you need to remove the register from database.*